tutorials/spherex/spherex_intro.md

```diff
@@ -93,12 +93,12 @@ The IRSA SIA collections can be listed using using the ``list_collections`` meth
 The collections are documented at [SPHEREx Data Access: Application Program Interfaces (APIs)](https://caltech-ipac.github.io/spherex-archive-documentation/spherex-data-access#application-program-interfaces-apis)
 There are currently three collections available:
```

Comment:
Are the old ones expected to go away? If they stay, they will show with list_collections, so it either needs a narrative update (as suggested) or some other indication that it's not only 3 available collections any more.
```suggestion
There are currently three collections available for the second Quick Release:
```